vector/src/main/res/values/strings.xml

```diff
@@ -3457,6 +3459,7 @@
     <string name="invite_people_to_your_space_desc">It’s just you at the moment.  %s will be even better with others.</string>
     <string name="invite_by_email">Invite by email</string>
     <string name="invite_by_mxid">Invite by username</string>
+    <string name="invite_by_mxid_or_mail">Invite by username or mail</string>
```

Comment:
The previous existing id `invite_by_mxid` was already wrong, so I understand why this one is wrong too :/

Reply:
Not sure to understand

Reply:
the id should be `invite_by_username`